File: app/src/main/java/com/yao/zhihudaily/model/CommentJson.java
package com.yao.zhihudaily.model;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

/**
 * @author Yao
 * @date 2016/8/30
 */
public class CommentJson {

    @SerializedName("comments")
    private ArrayList<Comment> comments;

    public ArrayList<Comment> getComments() {
        return comments;
    }

    public void setComments(ArrayList<Comment> comments) {
        this.comments = comments;
    }
}

File: app/src/main/java/com/yao/zhihudaily/model/CommentJson.kt
package com.yao.zhihudaily.model

import com.google.gson.annotations.SerializedName
import java.util.*

/**
 * @author Yao
 * @date 2016/8/30
 */
class CommentJson {

    @SerializedName("comments")
    var comments: ArrayList<Comment>? = null
}
